.github/workflows/release.workflow.yml

```
@@ -0,0 +1,26 @@
+name: Release workflow
+
+on: [release]
+
+jobs:
+  deploy:
+    name: Deploy
```

The job purpose is unclear to me, maybe we could rename it 
```suggestion
    name: Publish Package
```